src/lib/modernExtend.ts

```diff
@@ -1894,15 +1895,16 @@ function genericMeter(args?: MeterArgs) {
         if (args.producedEnergy !== false) exposes.push(e.produced_energy().withAccess(ea.STATE_GET));
         fromZigbee = [args.fzElectricalMeasurement ?? fz.electrical_measurement, args.fzMetering ?? fz.metering];
         toZigbee = [
-            tz.electrical_measurement_power,
+            args.reportingPowerType === "electrical" ? tz.electrical_measurement_power : tz.metering_power,
```

Comment:
Can't we base this off the `cluster` parameter?

Reply:
I don't have a good idea at the moment. I can't find a suitable parameter in the cluster. Do you have any good methods in mind? 
Thanks.

Reply:
Do you mean adding an argument such as 'both_meter' in the args.cluster parameter?

Reply:
I see what you want to do now, maybe we can do something like this?

```js
power?: false | (MultiplierDivisor & Partial<ReportingConfigWithoutAttribute> & {cluster?: "metering" | "electrical"});
```

Reply:
Oh, I see now. Thanks for clarifying. I will apply this immediately.